e2e/main.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/giantswarm/aws-operator/e2e/tests"
+)
+
+func main() {
+	// architect requires this
+	if len(os.Args) > 1 {
+		if os.Args[1] == "version" {
+			fmt.Println("0.1.0")
+			return
+		}
+
+		if os.Args[1] == "--help" {
+			fmt.Println("Yet another lookout")
```

Comment:
What is this logging for?

Reply:
For nothing in fact! :) thanks a lot for bringing this up, i just c&p'ed this from my hiring task :) Removed, much cleaner now